### PR TITLE
Share page component updates to new "𝕏" icon

### PR DESCRIPTION
**Pull request by adi-unni**
### What is the context of this PR?
Old component had the old twitter bird icon. New component has the new SVG for "𝕏" icon. Additional change has been made to fix alignment issues between the different icons. Now all icons are in line

### How to review this PR

Go to share page, and check icons are in line, and the new twitter "𝕏" icon can be seen.

### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

File: src/components/icon/_macro.njk [hunk 1]
```diff
@@ -72,8 +72,9 @@
...
+        <svg class="{% if params.iconSize %}ons-icon ons-icon--{{ params.iconSize }}{% else %}ons-icon{% endif %}{{ iconClasses }}" id="icon-twitter" viewBox="0 0 90.01 90.01" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
+            <polygon points="24.89,23.01 57.79,66.99 65.24,66.99 32.34,23.01 " style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;" transform="  matrix(1 0 0 1 0 0) "/>
+	        <path d="M 45 0 L 45 0 C 20.147 0 0 20.147 0 45 v 0 c 0 24.853 20.147 45 45 45 h 0 c 24.853 0 45 -20.147 45 -45 v 0 C 90 20.147 69.853 0 45 0 z M 56.032 70.504 L 41.054 50.477 L 22.516 70.504 h -4.765 L 38.925 47.63 L 17.884 19.496 h 16.217 L 47.895 37.94 l 17.072 -18.444 h 4.765 L 50.024 40.788 l 22.225 29.716 H 56.032 z" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;" transform=" matrix(1 0 0 1 0 0) " stroke-linecap="round" />
```
Comment (rmccar) [inline src/components/icon/_macro.njk:77]: Had a look at the SVG and removed a load of stuff we don't seem to need
```suggestion
        <svg class="{% if params.iconSize %}ons-icon ons-icon--{{ params.iconSize }}{% else %}ons-icon{% endif %}{{ iconClasses }}" id="icon-twitter" viewBox="0 0 90.01 90.01" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
            <polygon points="24.89,23.01 57.79,66.99 65.24,66.99 32.34,23.01"/>
            <path d="M 45 0 L 45 0 C 20.147 0 0 20.147 0 45 v 0 c 0 24.853 20.147 45 45 45 h 0 c 24.853 0 45 -20.147 45 -45 v 0 C 90 20.147 69.853 0 45 0 z M 56.032 70.504 L 41.054 50.477 L 22.516 70.504 h -4.765 L 38.925 47.63 L 17.884 19.496 h 16.217 L 47.895 37.94 l 17.072 -18.444 h 4.765 L 50.024 40.788 l 22.225 29.716 H 56.032 z"/>
        </svg>
```

**Review by rmccar — COMMENTED** [review]
Other than the SVG stuff looks good. Only question I have is do we change all references to "twitter" to "x"?

File: src/components/icon/_macro.njk (hunk 1 above)
Comment (adi-unni) [reply src/components/icon/_macro.njk]: I have updated the code to reflect the changes :)

**Comment by adi-unni** [comment]
Team has been consulted. Whilst twitter documentation is unclear and their marketing material is even more (the main page is still www.twitter.com but the page title is X and the logo is X), the decision has been made that the hyperlink text should be kept as twitter.

**Review by rmccar — APPROVED** [review]

**Review by precious-onyenaucheya-ons — APPROVED** [review]

**Pull request merged**